docroot/modules/humsci/hs_blocks/src/Plugin/Block/SocialMediaBlock.php

```diff
@@ -162,6 +165,31 @@ public function blockForm($form, FormStateInterface $form_state): array {
     return $form;
   }
 
+  /**
+   * Custom validation for the link_url field.
+   */
+  public static function validateUrl(array &$element, FormStateInterface $form_state, array &$complete_form) {
```

Comment:
can you use `\Drupal\Core\Render\Element\Url::validateUrl();` instead?

Reply:
Ok, changed.